core/commands/storage/helper/contracts.go

```diff
@@ -4,29 +4,35 @@ import guardpb "github.com/tron-us/go-btfs-common/protos/guard"
 
 var ContractFilterMap = map[string]map[guardpb.Contract_ContractState]bool{
 	"active": {
-		guardpb.Contract_DRAFT:    true,
-		guardpb.Contract_SIGNED:   true,
-		guardpb.Contract_UPLOADED: true,
-		guardpb.Contract_RENEWED:  true,
-		guardpb.Contract_WARN:     true,
+		guardpb.Contract_UPLOADED:  true,
+		guardpb.Contract_RENEWED:   true,
+		guardpb.Contract_WARN:      true,
+		guardpb.Contract_RECREATED: true,
```

Comment:
Recreated state mean the contract is submit from renter to guard, and waiting for guard to sign it  with host in the future, so it should not active contract. After it was signed with host, and it will be updated as Draft/Upload state

Reply:
ok, thats the same as draft state then, will remove it.